docs/pages/connect-your-client/vnet.mdx

```diff
@@ -79,19 +99,57 @@ If [per-session MFA](../admin-guides/access-controls/guides/per-session-mfa.mdx)
 first connection over each port triggers an MFA check.
 </Admonition>
 
-VNet is going to automatically start on the next Teleport Connect launch, unless you stop VNet
-before closing Teleport Connect.
+## Step 3/3. Connect to an SSH server
+
+VNet enables third-party SSH clients to connect to Teleport SSH servers with
```

Comment:
I would move this paragraph to the introduction of the guide, since it provides high-level information and links to other guides.